docs/features/duplicate_granule_handling.md

```diff
@@ -0,0 +1,60 @@
+---
+id: duplicate_granule_handling
+title: Cumulus Duplicate Granule Handling
+hide_title: false
+---
+
+Collections and Workflows can be configured to assign a unique `granuleId` to each Granule. This is useful in cases where two Granules share the same `granuleId` but must both be ingested into the Cumulus system. Previously, a Granule with a conflict on the `granuleId` key would cause an ingest failure for that Granule.
```

Comment:
`a Granule with a conflict on the `granuleId` key would cause an ingest failure for that Granule.` This is true for granuleId across collections, identical granuleId within the same collection will failure only when duplicateHandling is error.

Reply:
👍 Updated that section: https://github.com/nasa/cumulus/pull/4021/commits/5d24744de181c7ebbf52becf19919ba948cd5533